dashboard/lib/pd/foorm/workshop_summarizer.rb

```diff
@@ -158,5 +158,16 @@ def self.add_question_to_summary(summary_at_question, answer, answers, question_
       end
       summary_at_question
     end
+
+    # Sort summaries so that survey data is ordered in the order surveys
+    # should be taken in. i.e. Pre-Workshop, Day 1, Day 2,.. Post-Workshop
+    def self.sort_summary(workshop_summary)
+      temp_summary = workshop_summary.sort_by {|survey_key, _| get_index_for_survey_key(survey_key)}
+      workshop_summary_sorted = {}
+      temp_summary.each do |summaries|
+        workshop_summary_sorted[summaries[0]] = summaries[1]
```

Comment:
Could you comment on what this is doing?  I'm not quite sure at first glance..

Reply:
done!

Reply:
Thanks!